src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1965,10 +2022,10 @@ private String init(boolean initFull) {
                         selectedTemplate = testT;
                     }
                 }
-                workingVersion = dataset.getEditVersion(selectedTemplate, null);
+                workingVersion = dataset.getEditVersion(selectedTemplate, null, licenseServiceBean.getDefault());
```

Comment:
@qqmyers did some digging to find the reason I did this. The code where this default license could get set is here: [Dataset.java#L315-L322](https://github.com/IQSS/dataverse/blob/741f25520235101523e610f3e0849311b76b9220/src/main/java/edu/harvard/iq/dataverse/Dataset.java#L315-L322)

In the case that `latestVersion.getTermsOfUseAndAccess()` returns `null` the else part of the if is built to set a default license for TermsOfUseAndAccess. I ran into the problem here that I explained earlier, and that is getting the default license value from the database via an `@Entity` class. 

I could simply set the License to `null` here or remove the Else clause entirely, because I don't think TermsOfUseAndAccess can be `null` at this point anyway. Could be wrong

Reply:
Here's another example [DatasetVersion.java#L628-L638](https://github.com/IQSS/dataverse/blob/741f25520235101523e610f3e0849311b76b9220/src/main/java/edu/harvard/iq/dataverse/DatasetVersion.java#L628-L638) of an `@Entity` class that needs to initiate a default value from the database, the code previous to this feature could use a static reference to CC0, which is not the case anymore.

Reply:
@mderuijter - I've checked with QDR, Odum, and Harvard and none of them appear to have any versions with a null TermsOfUseAndAccess: `select * from datasetversion where termsofuseandaccess_id is null;` returns 0 rows. 
Given that, I think dropping the code for handling that case, which,  as we discussed, reduces the places you'd need the default license makes sense. 
(If anyone knows of any instance that does have db records where the termsofuseandaccess_id is null, we could look into an sql script/flyway update.)